### PR TITLE
Editorial widget

### PR DESCRIPTION
**Pull request by joanniclaborde**
https://github.com/SparkartGroupInc/jungle-solidus/wiki/Widgets#editorial

### CODE REVIEW

File: widgets/utils/wordpress.js [hunk 1]
```diff
@@ -39,3 +25,27 @@ module.exports.normalizeMedia = function(context, media) {
...
 };
+
+module.exports.normalizePost = function(context, post) {
+  if (!post) return null;
+
+  var result = {
+    topic:    post.slug,
+    title:    post.title,
+    tags:     module.exports.normalizeCategories(context, post.terms.category),
+    dateline: post.date == '-001-11-30T00:00:00+00:00' ? null : post.date,
```
Comment (pushred) [inline widgets/utils/wordpress.js]: What on earth is this!

Comment (joanniclaborde) [inline widgets/utils/wordpress.js]: I'd rather not display any date than to display this...

Comment (pushred) [inline widgets/utils/wordpress.js]: haha, where is it from though? Do we have a bad content import somewhere?

Comment (joanniclaborde) [inline widgets/utils/wordpress.js]: Oh, I have no idea, I don't remember where I saw this, I think it was a photo's date.

**Pull request merged**